src/render_engine/vulkan/vulkan_render_engine.cpp

```diff
@@ -141,6 +141,7 @@ namespace nova {
         depth_barriers.reserve(textures.size());
 
         for(const auto& [_, texture] : textures) {
+            (void) _;
```

Comment:
What and why is this?

Reply:
The code is using `_` to be representative of an unused value, but c++ doesn't special case `_` as an unused value. Therefore you must void cast it to tell the compiler to stfu. If you want to change this back to a normal variable name that's fine, but was out of scope of this PR.